### DISCUSSION TITLE
How to change a button back to the default color?

### DISCUSSION THREAD

**Discussion by RobertCordingly** [TomSchimansky/CustomTkinter#2458] [Q&A]
So I want to change a button's color when it's disabled and change it back when it's enabled again, but the obvious way to do this:

self.download_button.configure(fg_color=None)

Doesn't work because the button doesn't check with the ThemeManager when you call configure like it does when the button is initialized. 

How can I change the color back to the default?

**Comment by dipeshSam** [comment]
@RobertCordingly, you can utilise class `ThemeManager` of the `customtkinter`.

**Sample code:**
``` Python
from customtkinter import CTk, CTkButton
from customtkinter import ThemeManager


def get_color(class_name: str, property: str = "fg_color") -> tuple[str, str]:
    return ThemeManager.theme[class_name][property]

def on_click() -> None:
    if button._text != "Confirm":
        button.configure(fg_color = get_color("CTkButton"), text="Confirm", hover_color = get_color("CTkButton", "hover_color"))
    else:
        button.configure(fg_color = "brown", text="Cancel", hover_color="darkred")
        

if __name__ == "__main__":
    app = CTk()
    app.geometry("300x200")

    button = CTkButton(app, text="Confirm", command=on_click)
    button.place(relx=0.5, anchor="center", rely=0.5)


    app.mainloop()
```
**Output:**
![back_to_primary_color](https://github.com/TomSchimansky/CustomTkinter/assets/72375763/de755c01-6337-4e57-b0cd-3135e8a040f3)


Hope, this will be helpful.
Regards
